modules/_global-footer.hbs

```diff
@@ -0,0 +1,17 @@
+---
+title: "Global Footer"
+layout: module.hbs
+module: true
+---
+
+<!-- module:global-footer -->
+{{#parseFixture 'global-footer.json'}}
+<footer class="global-footer" id="global-footer" role="banner">
+
+	<span class="show-on-focus" name="global-footer">Footer</span>
```

Comment:
Josh had to change these to anchor tags for UCI because they were not working as spans. Something to consider?